src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/JavaScriptEncoder.cs

```diff
@@ -227,5 +228,10 @@ private static unsafe bool TryWriteEncodedSingleCharacter(int unicodeScalar, cha
             numberOfCharactersWritten = 6;
             return true;
         }
+
+        private static void ThrowHelper_ArgumentNullException(string paramName)
```

Comment:
This method is used only once, so you could move it to a local function and eliminate the `paramName` (i.e. hard-code `"text"`), so the code gets even smaller, as no string-argument must be created.

Reply:
Yes thanks.